omas/machine_mappings/d3d.json

```diff
@@ -113,32 +113,39 @@
  "coils_non_axisymmetric.coil.:.name": {
   "PYTHON": "coils_non_axisymmetric_hardware(ods, {pulse})"
  },
- "core_profiles.global_quantities.v_loop": {
-  "COCOSIO": 11,
-  "PYTHON": "core_profiles_global_quantities_data(ods, {pulse})"
- },
  "core_profiles.profiles_1d.:": {
-  "TDI": "size(\\{PROFILES_tree}::TOP.PROFILES.EDENSFIT,1)",
-  "treename": "{PROFILES_tree}"
+  "PYTHON": "core_profiles_profile_1d(ods, {pulse})"
  },
- "core_profiles.profiles_1d.:.electrons.density_thermal": {
-  "TDI": "\\{PROFILES_tree}::TOP.PROFILES.EDENSFIT",
-  "treename": "{PROFILES_tree}"
+ "core_profiles.profiles_1d.:.electrons.density": {
+  "PYTHON": "core_profiles_profile_1d(ods, {pulse})"
  },
  "core_profiles.profiles_1d.:.electrons.temperature": {
-  "TDI": "\\{PROFILES_tree}::TOP.PROFILES.ETEMPFIT",
-  "treename": "{PROFILES_tree}"
```

Comment:
Same here, but not because of errors but because of multiple sources for the same field. I don't know how to resolve this.

Reply:
@orso82 How does one specify in the machine mappings when the same quantity is available via a variety of MDSplus trees?  Maybe passing an argument to `ods.open` that gets passed to the underlying new `core_profiles_profile_1d`?